extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -78,4 +73,26 @@ public TransferHistoryPage transferHistoryPageEndpoint() {
     public AssetDto getTransferProcessAsset(@PathParam("transferProcessId") String transferProcessId) {
         return transferHistoryPageAssetFetcherService.getAssetForTransferHistoryPage(transferProcessId);
     }
+
+    @GET
+    @Path("pages/asset")
+    @Produces(MediaType.APPLICATION_JSON)
+    public AssetPage assetPage() {
+        return new AssetPage(assetApiService.getAssets());
+    }
+
+    @POST
+    @Path("pages/asset")
+    @Produces(MediaType.APPLICATION_JSON)
+    public IdResponse createAsset(AssetRequest assetRequest) {
+        return assetApiService.createAsset(assetRequest);
+    }
+
+    @DELETE
+    @Path("pages/asset")
+    @Produces(MediaType.APPLICATION_JSON)
+    public IdResponseDto deleteAsset(String assetId) {
```

Comment:
This should come via a path parameter. As we use the generated API clients we notice these differences less, we should still focus on correct API design following some convention / standards

`@PathParam("assetId") String assetId`